x-pack/plugins/actions/server/create_unsecured_execute_function.ts

```diff
@@ -22,13 +28,21 @@ interface CreateBulkUnsecuredExecuteFunctionOptions {
   taskManager: TaskManagerStartContract;
   connectorTypeRegistry: ConnectorTypeRegistryContract;
   inMemoryConnectors: InMemoryConnector[];
+  configurationUtilities: ActionsConfigurationUtilities;
 }
 
 export interface ExecuteOptions
   extends Pick<ActionExecutorOptions, 'params' | 'source' | 'relatedSavedObjects'> {
   id: string;
 }
 
+export type ExecutionResponseItem = Pick<ActionExecutionResponseItem, 'id' | 'response'>;
+
+export interface ExecutionResponse {
```

Comment:
is it possible to reuse the same type as used in `create_execute_response`?

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/164632/commits/40f2675ba795fa8e8e8fb911bafd260de3f57818